ItemRepeater.cs

```diff
@@ -109,7 +109,10 @@ public override bool Run(IssueManager issueManager)
                             {
                                 try
                                 {
-                                    field.Entity.Id = 0;
+                                    //Find the existing ID to 'replace', if exists.
```

Comment:
Since IssueManager.Create() now creates all the custom fields, they are already existing on the new IssueDto. Updating with an Id=0 will create a new field, hence the duplicates. Now, it finds the ID in the custom fields and uses that ID so it is updated (and the value will be on the new item). If nothing found, will use 0 and create the CF